TESTS.md

```diff
@@ -1340,6 +1340,24 @@ data D =
                   c
 ```
 
+sophie-h Regression: Breaks basic type class code by inserting "|" #459
+
+```haskell
+-- https://github.com/commercialhaskell/hindent/issues/459
+class Class1 a =>
+      Class2 a
+  where
```

Comment:
Should `where` be on a new line? I think in 5.2.3 it was on the same line as `Class2 a`.

Reply:
Indeed there's also an extra new line in version 5.2.3. It's a bug and will be fixed in next release.